sdk/openai/openai/samples/cookbook/speechToSpeechChat/README.md

```diff
@@ -0,0 +1,15 @@
+This is a browser page demonstrating speech to speech chat implementation using the Azure OpenAI JS SDK and Azure JS Speech SDK.
```

Comment:
How is this different than the built-in browser [Speech API](https://developer.mozilla.org/en-US/docs/Web/API/SpeechRecognition) and why should I use Azure here?

Reply:
1. The JS Azure Speech SDK produces best-in-class recognition results
2.  A fair number of Azure customers are already using Azure Speech SDK
3.  This sample is for parity with the python samples, which similarly use Azure Speech SDK

This isn't considering the business value, but we can followup with the PMs (Arun, Travis, and Rob Chambers) if you'd like.

Reply:
Yes, that would be ideal given that there is already a native API for this.

Reply:
@mpodwysocki After talking with @deyaaeldeen, I've added a checkbox to allow use of the Web Speech API, with example code using that API as well. Let me know what you think.

Reply:
@glharper sounds good